packages/web/package.json

```diff
@@ -15,12 +15,16 @@
   "main": "js/cjs/spirit-web.js",
   "module": "js/esm/spirit-web.js",
   "scripts": {
+    "icons:build": "shx mkdir -p public/icons && shx cp -r ../../node_modules/@lmc-eu/spirit-icons/dist/. public/icons",
+    "prestart": "shx mkdir -p public/icons && cd ../icons && yarn build && shx cp -r dist/. ../web/public/icons",
     "start": "vite --host",
+    "preexamples:build": "yarn icons:build",
     "examples:build": "vite build",
     "examples:build:gh": "yarn examples:build --base=/spirit-design-system/",
     "examples:preview": "vite preview",
     "prebuild": "shx rm -rf dist && shx mkdir -p dist/scss && shx cp package.json README.md dist/ && shx cp -r src/scss dist/",
     "build": "npm-run-all --serial css js",
+    "postbuild": "yarn icons:build",
```

Comment:
Why do we need this `postbuild` and copying of icons? They are not needed in `./dist` and also not there.